README.md

```diff
@@ -18,14 +18,16 @@ To run the REPL just enter the following in the source code directory
 
 and start typing REPL commands there. The syntax of the REPL commands is
 
-    <repl-command> := <term> | <binding>
+    <repl-command> := <term> | <binding> | <loading>
     <term> ::= <variable>
              | <numeric-literal>
              | <character-literal>
              | <string-literal>
              | <function>
              | <application>
     <binding> ::= <variable> := <term>
+    <loading> ::= load <module-path>
+    <module-path> := [a-zA-Z][a-zA-Z0-9/]*
```

Comment:
I am sure it would be better to define `module-path` as `string-literal`. Your current definition doesn't supports paths with spaces or cyrillic (greek?) letters, that's not nice!